Fixed typos, names, minor style improvements (#156)
Mirantis/mke-docs: content/docs/configuration/telemetry.md

helps product teams in their efforts to enhance Mirantis products and
services.

{{< callout type="info" >}}
   Telemetry is automatically enabled for MKE 4 clusters that are running
   without a license, with a license that has expired, or with an invalid

>    license. In all of such scenarios, you will only be able to disable

Reviewer: ```suggestion
   license. In all of such scenarios, you can only disable
```
